### PR TITLE
Add Batch Normalization example to README

### PR DESCRIPTION
**Pull request by shyam-vyawahare**
Added a Batch Normalization example using PyTorch to the deep learning section.

### CODE REVIEW

**Review by shyam-vyawahare — COMMENTED** [review]
File Structure updates
